README.md

```diff
@@ -2,11 +2,58 @@ API-Beheer
 =====
 [![Build Status](https://jenkins.nlx.io/job/gemma-zaken-build-and-test/badge/icon?style=plastic)](https://jenkins.nlx.io/) ![Repo Status](https://img.shields.io/badge/status-concept-lightgrey.svg?style=plastic)
 
-**API-Beheer** 
-Visie
-
 ## Doel
-Deze repository bevat alles wat nodig is...
+
+Deze repository bevat alles wat nodig is om de GEMMA ZDS 2.0 Open API standaard in beheer te kunnen nemen en om dat beheer goed uit te kunnen voeren.
+
+> _**Nog te beantwoorden vragen:**_
+> - _Is er nog wel sprake van de GEMMA ZDS 2.0 Open API standaard? Theo spreekt liever over de zaakgerichte standaarden wat impliceert dat er meerdere standaarden zijn. Als dat zo is, is het doel van dit project dan de in beheername van al deze standaarden of alleen van een van die standaarden?_
```

Comment:
Ik zou eerder zeggen: De ZDS 2.0 standaard is geheel van standaarden voor het zaakgericht werken.  En dan de vraag over het doel van dit project